docs/spec/relayer/Data-Requirements.md

```diff
@@ -162,3 +162,30 @@ The relayer connects to the node's CometBFT websocket interface and subscribes t
     - `EXISTS acknowledge_packet`
     - `EXISTS timeout_packet`
 
+## CometBFT gRPC
+
+The following are the gRPC endpoints that Hermes makes use of, listed in rough priority from most used to least used. 
+
+### `ibc.core.channel.v1.QueryClient`
```

Comment:
On top of listing the client/service used to perform the query, can we also list the actual queries that are being made?

eg. for `ibc.core.channel.v1.QueryClient` that would be:
- `QueryChannelClientStateRequest`
- `QueryChannelsRequest`
- `QueryConnectionChannelsRequest`
- `QueryNextSequenceReceiveRequest`
- `QueryPacketAcknowledgementsRequest`
- `QueryPacketCommitmentsRequest`
- `QueryUnreceivedAcksRequest`
- `QueryUnreceivedPacketsRequest`